app/ModelSerializers/Summit/Presentation/TrackChairPresentationSerializer.php

```diff
@@ -29,6 +29,7 @@ class TrackChairPresentationSerializer extends AdminPresentationSerializer
         'VotesCount'      => 'votes_count:json_int',
         'VotesAverage' => 'votes_average:json_float',
         'VotesTotalPoints' => 'votes_total_points:json_int',
```

Comment:
its missing the AVG os the presentation
please add following attribute getTrackChairAvgScore like float (track_chair_avg_score:json_float)

Reply:
done